docs/examples.rst

```diff
@@ -0,0 +1,40 @@
+Examples
+============
+
+SchemaRegistry comes with examples of integration into Kafka. You can find the code
+here https://github.com/hortonworks/registry/tree/master/examples/schema-registry/avro/src/main/java/com/hortonworks/registries/schemaregistry/examples
+
+
+Running Kafka Producer with AvroSerializer
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Login into one of the Kafka broker hosts
+2. bin/kafka-topics.sh --zookeeper zk:2181 --topic truck_events_stream --partitions 2 --replication-factor 1 --create
+3. On registry host; cd /usr/hdf/2.1.0.0-164/registry/examples/schema-registry/avro/
+4. Edit data/kafka-producer.props, vi data/kafka-producer.props
+
+topic=truck_events_stream
+
+bootstrap.servers=kafka_host1:6667,kafka_host2:6667
+
+schema.registry.url=http://regisry_host:9090/api/v1
+
+
+5. java -jar avro-examples-0.2.jar -d data/truck_events.csv -p data/kafka-producer.props -sm -s data/truck_events.avsc
+6. The above command will register truck_events schema in data/truck_events.avsc into registry and ingests 200 messages into topic “truck_events_stream”
+
+
+To run the producer in Secure cluster:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Issue ACLs on the topic you are trying to ingest
+
+2. create kafka topic
+   bin/kafka-acls.sh --authorizer kafka.security.auth.SimpleAclAuthorizer --authorizer-properties zookeeper.connect=pm-eng1-cluster2.field.hortonworks.com:2181 --add
+       --allow-principal User:principal_name --allow-host "*" --operation All --topic truck_events_stream
+
+   2.1 make sure you replace "principal_name" with the username you are trying to ingest
+
+3.  edit data/kafka-producer.props , add security.protocol=SASL_PLAINTEXT
+
+4. java -Djava.security.auth.login.config=/etc/kafka/conf/kafka_client_jaas.conf -jar avro-examples-0.3.0-SNAPSHOT.jar -d data/truck_events_json -p data/kafka-producer.props -sm -s data/truck_events.avsc
```

Comment:
Assuming that jaas config file is from Ambari installation, it doesn't contain principal and keytab so need to kinit with proper principal before executing command. I was doing same to test Streamline with secure cluster.